src/services/api.ts

```diff
@@ -12,14 +12,26 @@ export const getInnholdsinformasjon = async (isbn: string): Promise<Innholdsform
     return innholdsinformasjon;
   }
   innholdsinformasjon.title = innholdResponse.title;
-  innholdsinformasjon.image_small = innholdResponse.image_small;
+  innholdsinformasjon.image_path = getImagePath(
+    innholdResponse.image_small,
+    innholdResponse.image_original,
+    innholdResponse.image_large
+  );
   innholdsinformasjon.description_short = splitOnTags(innholdResponse.description_short).map(removeAllTags);
   innholdsinformasjon.description_long = splitOnTags(innholdResponse.description_long).map(removeAllTags);
   innholdsinformasjon.table_of_contents = splitOnTags(innholdResponse.table_of_contents).map(removeAllTags);
 
   return innholdsinformasjon;
 };
 
+function getImagePath(small: string, original: string, large: string) {
+  if (small) {
+    return small;
+  } else {
+    return original ? original : large;
+  }
+}
+
 function removeAllTags(value: string) {
```

Comment:
Pirk: Anbefaler å holde seg til en naming-convention gjennom hele prosjektet:
const removeAllTags = (value: string) =>  { 
}

De siste måten å skrive funksjoner på sørger også for at funksjonen ikke tar i bruk variabler i det globale skopet

Reply:
Gjelder alle funksjonene i denne fila